python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -164,8 +202,12 @@ def __init__(self, era, algo = 'csvv2', sfFileName = None, verbose = 0):
         self.branchNames_central_and_systs = {}
         for central_or_syst in self.central_and_systs:
             if central_or_syst == "central":
+                if era.find("FastSim"):
+                    self.branchNames_central_and_systs[central_or_syst] = "Jet_btagSF_FS"
```

Comment:
How do people use this?

Reply:
If you look on the NanoSUSY-tools command on the stop0l_postproc file the FastSim module listed below adds in the btagging module which is where the era is added in. Here is the line: 
btagSFProducer(args.era+"FastSim", algo="deepcsv"),
And the lines in the branch:
https://github.com/susy2015/NanoSUSY-tools/blob/dev_v5_FastSim_Btagging/python/processors/Stop0l_postproc.py#L238-L242